### DISCUSSION TITLE
Magento Custom Form Builder

### DISCUSSION THREAD

**Discussion by sandipansaha** [breezefront/community#113] [3. Integrations]
Magento Custom Form Builder
https://amasty.com/custom-form-for-magento-2.html
or
https://mirasvit.com/magento-2-form-builder.html

**Comment by dimdimich** [comment]
Thank you for you suggestion. Please let us know if you can contact both Mirasvit and Amasty and ask for integration. Recently we got more and more requests on integration. So simpply letting Amasty and Mirasvit know that more people are using their modules with Breeze  will help them to understand that it's good decision to add Breeze support from their side.
